tests/integrate_test/test_integrate_scores.py

```diff
@@ -28,6 +28,21 @@
 
 class TestIntegrateScores(TestIntegrateBase):
 
+    def _update_governance(self):
```

Comment:
You don't need to modify this test case if you apply my suggestion above.
Please revert the change of this file.  This should be remained for regression tests.

Reply:
got it.
I'll keep that in mind